R/auth_sitrep.R

```diff
@@ -0,0 +1,187 @@
+#' Twitter Tokens sitrep
+#' 
+#' Get a situation report of your current tokens; useful for upgrading from 
```

Comment:
It appears to move the tokens around and create directories, which really doesn't fit into the framework of a "sitrep" function. I'd suggest a different name.

Reply:
Yes, that would be a good idea.